universal-application-tool-0.0.1/app/views/applicant/TrustedIntermediaryDashboardView.java

```diff
@@ -80,18 +89,67 @@ public Content render(
     return layout.renderWithNav(request, userName, messages, bundle);
   }
 
-  private ContainerTag renderTIApplicantsTable(TrustedIntermediaryGroup tiGroup) {
-    return div(
-        table()
-            .withClasses(Styles.BORDER, Styles.BORDER_GRAY_300, Styles.SHADOW_MD, Styles.W_3_4)
-            .with(renderApplicantTableHeader())
-            .with(
-                tbody(
-                    each(
-                        tiGroup.getManagedAccounts().stream()
-                            .sorted(Comparator.comparing(Account::getApplicantName))
-                            .collect(Collectors.toList()),
-                        account -> renderApplicantRow(account)))));
+  private ContainerTag renderSearchForm(Http.Request request, Optional<String> search) {
+    return form()
+        .withMethod("GET")
+        .withAction(
+            routes.TrustedIntermediaryController.dashboard(Optional.empty(), Optional.empty())
+                .url())
+        .with(
+            FieldWithLabel.input()
+                .setId("search-field")
+                .setFieldName("search")
+                .setLabelText("Search")
+                .setValue(search.orElse(""))
+                .setPlaceholderText("Search")
+                .getContainer()
+                .withClasses(Styles.W_1_4),
+            makeCsrfTokenInputTag(request),
+            submitButton("Search").withClasses(Styles.M_2));
+  }
+
+  private ContainerTag renderTIApplicantsTable(
+      ImmutableList<Account> managedAccounts,
+      Optional<String> search,
+      int page,
+      int totalPageCount) {
+    ContainerTag main =
+        div(table()
+                .withClasses(Styles.BORDER, Styles.BORDER_GRAY_300, Styles.SHADOW_MD, Styles.W_3_4)
+                .with(renderApplicantTableHeader())
+                .with(
+                    tbody(
+                        each(
+                            managedAccounts.stream()
+                                .sorted(Comparator.comparing(Account::getApplicantName))
+                                .collect(Collectors.toList()),
+                            account -> renderApplicantRow(account)))))
+            .withClasses(Styles.MB_16);
+    ContainerTag div = div();
+    if (page <= 1) {
+      div.with(new LinkElement().setText("∅").asButton());
+    } else {
+      div.with(
+          new LinkElement()
+              .setText("←")
+              .setHref(
+                  routes.TrustedIntermediaryController.dashboard(search, Optional.of(page - 1))
+                      .url())
+              .asButton());
+    }
+    div.with(new LinkElement().setText("Page " + page + " of " + totalPageCount).asButton());
```

Comment:
Can we convert this to plain text?

Reply:
I tried - the alignment goes all screwy and I can't figure out how to fix it.  This is the only trick I could figure out (even though it looks silly).  What classes would you apply to a `div` (or a `p`?  something else?) to get it to flow / align like one of these buttons?  I could give that a shot.